package.json

```diff
@@ -68,6 +68,7 @@
   "dependencies": {
     "@rollup/plugin-json": "^4.1.0",
     "babel-eslint": "^10.1.0",
+    "classnames": "^2.3.1",
```

Comment:
@yousseefben  with styled components there is no need for classnames - see other components how they solve active state and such

Reply:
@MoritzGruber yes i know, i think that's a better way with adding classnames, but it's okay i'll follow your guides, i will push the code right now